import-mailbox-to-gmail.py

```diff
@@ -116,11 +116,9 @@ def get_credentials(username):
   Returns:
     Credentials, the obtained credential.
   """
-  json_file = json.load(open(args.json))
-  credentials = SignedJwtAssertionCredentials(json_file['client_email'],
-                                              json_file['private_key'],
-                                              SCOPES,
-                                              sub=username)
+  credentials = ServiceAccountCredentials.from_json_keyfile_name(
+          args.json,
```

Comment:
Please make the leading spaces consistent with the rest of the file, i.e. use 4 instead of 8.